src/WebJobs.Script/Binding/NotificationHubScriptBindingProvider.cs

```diff
@@ -51,12 +51,8 @@ public override bool TryResolveAssembly(string assemblyName, out Assembly assemb
         {
             assembly = null;
 
-            if (string.Compare(assemblyName, "Microsoft.Azure.NotificationHubs", StringComparison.OrdinalIgnoreCase) == 0)
```

Comment:
We have built in logic that will automatically add the assembly where the ScriptBindingProvider was from. However in https://github.com/Azure/azure-webjobs-sdk-script/commit/2b3b5b53f046c99dc9077f6ba28aed1c56c91f3f I moved all of them into this repo. I had to do that release 2.0 Extensions - non-prerelease packages cannot reference prerelease packages.